rootex/core/resource_files/material_resource_file.h

```diff
@@ -71,12 +71,19 @@ struct PerModelVSCBData
 {
 	Matrix model;
 	Matrix modelInverseTranspose;
+	PerModelPSCBData material;
```

Comment:
this is semantically wrong. Also, I don't think we need perModelPSCB data, I guess you can write the contents of the perModelPSCBData in this itself. You can even try to optimise padding.